products/models.py

```diff
@@ -5,9 +5,22 @@
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 
+from products.constants import IMAGES_PATH
 from users.models import TimeStamp, User
 
 
+def image_path(instance, filename):
+    """
+    Returns the path to store the image file at
+    Args:
+        instance(Product): Value containing product data
+        filename(str): Value containing image filename
+    Returns:
+        (str): Value containing location to store file
+    """
+    return f"{IMAGES_PATH}{instance.name}_{filename}"
```

Comment:
This looks like something we might use as a utility. Can we move this to some utility file?

Reply:
Sure, Arslan bhai, but i do not see any future use of this function anywhere else. Should we move functions to utils if they have no usage elsewhere? Please let me know.

Reply:
Do you see the future usage in the account/profile page for a profile pictures?

Reply:
Yes, thank you for clarifying.